website/static/js/citationGrid.js

```diff
@@ -377,11 +367,11 @@ CitationGrid.prototype.makeBibliography = function(folder, format) {
     return {};
 };
 
-CitationGrid.prototype.getBibliography = function(folder, format) {
```

Comment:
Are there any calls to this function that attempt to pass `format` that need to be updated?

Reply:
Nothing breaking but I removed the parameter where it was unnecessary from other functions.